Formula/vis.rb

```diff
@@ -38,12 +38,16 @@ def install
       system "luarocks", "build", "lpeg", "--tree=#{luapath}"
     end
 
-    system "./configure", "--prefix=#{libexec}"
+    system "./configure", "--prefix=#{prefix}"
     system "make", "install"
+
+    # Rename vis & the matching manpage to avoid clashing with the system.
+    man1.install man1/"vis.1" => "vise.1"
+    (libexec/"bin").install bin/"vis"
     (bin/"vise").write <<~EOS
       #!/bin/sh
-      LUA_PATH="#{ENV["LUA_PATH"]}"
-      LUA_CPATH="#{ENV["LUA_CPATH"]}"
+      export LUA_PATH="#{ENV["LUA_PATH"]}"
+      export LUA_CPATH="#{ENV["LUA_CPATH"]}"
       VIS_BASE=#{libexec}
```

Comment:
Is this still correct if we use prefix not libexec?

Reply:
No, that's a screw up on my part.